scripts/accumulator.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/716d723547fc7212c70b4612441ac483ab055cdd